package.json

```diff
@@ -17,56 +17,52 @@
   "homepage": "https://videojs.com",
   "author": "Steve Heffernan",
   "scripts": {
-    "test:node-require": "node test/require/node.js",
-    "build:browserify-test": "browserify test/require/browserify.js -o build/temp/browserify.js",
-    "build:webpack-test": "webpack --hide-modules test/require/webpack.js build/temp/webpack.js",
-    "copy-fonts": "shx cp -R node_modules/videojs-font/fonts build/temp/font",
-    "copy-dist": "shx cp -R build/temp/* dist/",
-    "copy-examples": "shx cp -R docs/examples dist/",
+    "start": "npm-run-all -p watch karma-server",
+    "clean": "shx rm -rf ./dist ./test/dist ./docs/api",
+    "postclean": "shx mkdir -p ./dist/lang ./test/dist",
+    "changelog": "conventional-changelog -p videojs -i CHANGELOG.md -s",
+    "build": "npm-run-all build-dev minify zip",
+    "build-dev": "npm-run-all clean lint-errors build:js build:css build:lang build:copy build:test",
+    "build:test": "npm-run-all build:test:*",
+    "build:test:a11y": "shx cp sandbox/descriptions.html.example sandbox/descriptions.test-a11y.html",
+    "build:test:browserify": "browserify test/require/browserify.js -o test/dist/browserify.js",
+    "build:test:webpack": "webpack --hide-modules test/require/webpack.js test/dist/webpack.js",
+    "build:copy": "npm-run-all build:copy:*",
```

Comment:
I would make these just `copy` and `copy:fonts`, etc

Reply:
what if we just changed it to `build:fonts` and `build:examples` instead? To me even though these are not really building anything, they need to run with the rest of the build.

Reply:
I guess they could run as `copy` with `build` instead of `build-dev`

Reply:
I don't think either fonts or examples are used anywhere directly except for providing those files in the published tarball